RedHatStyleGuide/en-US/rel-notes.xml

```diff
@@ -32,5 +36,11 @@
 			<para>Minor edits so the guide itself conforms with its own advice.</para>
 
 	</section>
+	 <section id="sect-Red_Hat_Technical_Writing_Style_Guide-Older_Release_Notes">
+		<title>Earlier Releases than 5.0</title>
```

Comment:
"Releases Prior to 5.0"?

Reply:
I thought our standard was to use "earlier" or "later" to refer to releases?

Reply:
Yeah I know what you're saying, but in this construction it just sounds weird. If it were "In release 5.0 or earlier" then sure, no problem. Not a hill to die on in this case IDT.

Reply:
How about "Releases Earlier than 5.0"?

Reply:
Are we getting hung up on "Thou shalt use 'earlier' and 'later'" at the expense of creating comprehensible English? 

First words in the Inquirer style book: "Any rule can be overruled by 1) using common sense and 2) trusting your ear."

So, I'll call this approved and rely on your judgement.

Reply:
Applied final wording changes.